ui/app/components/project-input-variables/list-item.hbs

```diff
@@ -1,116 +1,139 @@
 <li data-test-input-variables-list-item>
   {{#if (or this.isEditing this.isCreating)}}
-    <form class="pds-form" data-test-input-variables-form {{on "submit" this.saveVariable}}>
```

Comment:
(suggestion) Looks like we don't enforce it in linting tests, but perhaps it would be better to remove the quote changing in this file, to match the rest of the repo, which uses `"`. 

(I think `"` is the Ember default in `.hbs`, but `cloud-ui` uses `'`.) Maybe you have a VSCode extension doing an autosave?

Reply:
Added changelog.